examples/advanced/quickxplain.py

```diff
@@ -0,0 +1,83 @@
+"""
+    CPMpy implementation of the QuickXplain algorithm by Junker.
+        Junker, Ulrich. "Preferred explanations and relaxations for over-constrained problems." AAAI-2004. 2004.
+"""
```

Comment:
Add url of paper because it's hard to find: 
https://cdn.aaai.org/AAAI/2004/AAAI04-027.pdf